locales/en-US/translation.json

```diff
@@ -1058,7 +1058,7 @@
     "1": "A friendly reminder you're using the {{testnet}} network build - the balances are not real"
   },
   "dollarBalance": "<0></0> $t(celoDollars)",
-  "feedItemAddress": "Account {{address}}",
```

Comment:
nit: we should only update the base translation as it can cause merge issues for other translations since they are handled by Crowdin.